pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/JavaInstanceMain.java

```diff
@@ -82,7 +84,25 @@
 
     @Parameter(names = "--pulsar_serviceurl", description = "Pulsar Service Url\n", required = true)
     protected String pulsarServiceUrl;
-
+    
+    @Parameter(names = "--client_auth_plugin", description = "Client auth plugin name\n")
+    protected String clientAuthenticationPlugin;
+    
+    @Parameter(names = "--client_auth_params", description = "Client auth param\n")
+    protected String clientAuthenticationParameters;
+    
+    @Parameter(names = "--use_tls", description = "Use tls connection\n")
+    protected boolean useTls;
+    
+    @Parameter(names = "--tls_allow_insecure", description = "Allow insecure tls connection\n")
+    protected boolean tlsAllowInsecureConnection;
+    
+    @Parameter(names = "--hostname_verification_enabled", description = "Enable hostname verification")
+    protected boolean tlsHostNameVerificationEnabled;
```

Comment:
I think `useTls`, `tlsAllowInsecureConnection` and `tlsHostNameVerificationEnabled` are Strings, not booleans.
https://github.com/rdhabalia/pulsar/blob/3b6fbaeaec5c96c12091d6e839e04c297c208caa/pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/ProcessRuntime.java#L152-L157
This causes ParameterException:
```
Exception in thread "main" org.apache.pulsar.functions.runtime.shaded.com.beust.jcommander.ParameterException: Was passed main parameter 'false' but no main parameter was defined
        at org.apache.pulsar.functions.runtime.shaded.com.beust.jcommander.JCommander.getMainParameter(JCommander.java:914)
        at org.apache.pulsar.functions.runtime.shaded.com.beust.jcommander.JCommander.parseValues(JCommander.java:759)
        at org.apache.pulsar.functions.runtime.shaded.com.beust.jcommander.JCommander.parse(JCommander.java:282)
        at org.apache.pulsar.functions.runtime.shaded.com.beust.jcommander.JCommander.parse(JCommander.java:265)
        at org.apache.pulsar.functions.runtime.JavaInstanceMain.main(JavaInstanceMain.java:289)
```

Reply:
yes, not sure, why we are converting to string.. will get back to this.. but I have fixed it for now.. thanks for catching it..